design/ceph/object/store.md

```diff
@@ -119,3 +119,39 @@ Specifying this section also ensures that the pool section in the ceph-object-zo
   zone:
     name: "name"
 ```
+
+### LDAP
+
+The Ceph Object Gateway supports integrating with LDAP for authenticating and creating users, please refer [here](https://docs.ceph.com/docs/master/radosgw/ldap-auth/). This means that the `ceph object store user` is also required to be part of groups in the LDAP server otherwise, authentication will fail. The following settings need to be configured in the RGW server:
+```
+rgw ldap binddn =
+rgw_ldap_secret = /etc/ceph/ldap/bindpass.secret
+rgw ldap uri =
+rgw ldap searchdn =
+rgw ldap dnattr =
+rgw ldap searchfilter =
+rgw_s3_auth_use_ldap = true
+```
+So the CRD for the Ceph Object Store will be modified to include the above changes:
+```yaml
+ldap:
```

Comment:
Let's figure out where the ldap section should go in the CRD. for CephNFS, it is under the `security` heading. I'd propose this be `spec.security.ldap`

```suggestion
spec:
  security:
    ldap:
```

Reply:
This sounds reasonable to me.